static/app/components/events/interfaces/frame/deprecatedLine.tsx

```diff
@@ -1,4 +1,4 @@
-import {Component} from 'react';
+import React, {Component} from 'react';
```

Comment:
nit we prefer just importing fragment instead of everything and using react.fragment
```suggestion
import {Component, Fragment} from 'react';
```